components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -231,95 +267,202 @@ public function auto_pods() {
 	}
 
 	/**
-	 * Fetches the current post type.
+	 * Get the filter used for a Pod.
+	 *
+	 * @param string $pod_name      The pod name.
+	 * @param array  $possible_pods Array of available pods. See self::auto_pods().
+	 *
+	 * @return string
+	 * @since  1.7.2
+	 */
+	public function get_pod_filter( $pod_name = '', $possible_pods = array() ) {
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: a86685e676b7d7da1c8528487ed03d05</sub>